src/com/reason/Compiler.java

```diff
@@ -12,6 +17,11 @@
 
 public interface Compiler {
 
+    static boolean isSupportedFileType(@NotNull FileType fileType) {
+        return fileType instanceof OclFileType || fileType instanceof OclInterfaceFileType
+                || fileType instanceof RmlFileType || fileType instanceof RmlInterfaceFileType;
+    }
```

Comment:
use FileHelper.isOcaml || FileHelper.isReason ?

Reply:
didn't know this existed. thanks.